src/components/Table/Table.tsx

```diff
@@ -143,11 +171,30 @@ export const Table: ChakraComponent<
               <ChakraTr key={index}>
                 {row.map((column, key) =>
                   key === 0 && useRowHeaders ? (
-                    <ChakraTh scope="row" key={key}>
+                    // row header cell
+                    <ChakraTh
+                      scope="row"
+                      key={key}
+                      sx={
+                        isScrollable
+                          ? columnStyles[key]
+                          : { base: undefined, md: columnStyles[key] }
```

Comment:
I think this also works if you leave out the `base` property, right?

Reply:
Tthis code actually does not work as expected, regardless of using `base` or not.  I had to rewrite it using `isLargerThanMobile`, but I am concerned about possible style flickering.